samples/chatbot-easy-rag/pom.xml

```diff
@@ -2,15 +2,36 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>io.quarkiverse.langchain4j</groupId>
-        <artifactId>quarkus-langchain4j-parent</artifactId>
-        <version>999-SNAPSHOT</version>
-        <relativePath>../../pom.xml</relativePath>
-    </parent>
-
+    <groupId>io.quarkiverse.langchain4j</groupId>
     <artifactId>quarkus-langchain4j-sample-chatbot-easy-rag</artifactId>
     <name>Quarkus LangChain4j - Sample - Chatbot with Easy RAG</name>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <compiler-plugin.version>3.12.1</compiler-plugin.version>
+        <maven.compiler.parameters>true</maven.compiler.parameters>
+        <maven.compiler.release>17</maven.compiler.release>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
```

Comment:
Any reason to not have it use the platform as is recommended for user apps?

Reply:
The only reason is that I can easily switch to snapshot Quarkus when needed 😎

Reply:
+1, I tackle this problem often, having to also build a snapshot of the platform would be painful

Reply:
In Quarkus quick start it's a property so easy to override. Snapshot/Quarkus can be in a profile. 

Just that it would be nice the samples works as defacto standard Quarkus app and allow quarkus ext add work like every other default app.